src/test/java/cz/muni/pa165/teamwhite/formula1/DriverTest.java

```diff
@@ -0,0 +1,266 @@
+package cz.muni.pa165.teamwhite.formula1;
```

Comment:
```suggestion
package cz.muni.pa165.teamwhite.formula1.dao;
```

Move to .dao package

Reply:
fixed